buildout.cfg

```diff
@@ -215,4 +215,4 @@ recipe = collective.recipe.cmd
 on_install = true
 on_update = true
 cmds =
-    curl -o annotations.json https://s3-us-west-1.amazonaws.com/encoded-build/annotations/annotations_2017_10_10.json
+    curl -o annotations.json https://s3-us-west-1.amazonaws.com/encoded-build/annotations/annotations_2019_07_11.json
```

Comment:
This doesn't look related to the ticket.  Why is an annotations update in this pr?

Reply:
@caseylitton I updated the name to signify the file is updated (despite time stamp); to follow the apparent convention. I can keep the current name if you like.

Reply:
@caseylitton the bug that appears in ENCD-3904 is caused by extra " in the file that is created by the script.  So this PR does both; fixes the script (writes a new file) and corrects the download of the fixed file.

Reply:
Are you sure?  The two annotation files are the exact same

annotations_2017_10_10.json
annotations_2019_07_11.json

Reply:
@caseylitton it turns out the old file was "illegally" updated recently... we will restore it.

Reply:
I updated "annotations_2017_10_10.json" to be the same content as prior. I could not revert as that was no enabled before.